Gems/Atom/Feature/Common/Code/Source/CoreLights/DirectionalLightFeatureProcessor.cpp

```diff
@@ -207,12 +210,21 @@ namespace AZ
             {
                 SetFullscreenPassSettings();
 
-                const uint32_t shadowFilterMethod = m_shadowData.at(nullptr).GetData(m_shadowingLightHandle.GetIndex()).m_shadowFilterMethod;
+                uint32_t shadowFilterMethod = m_shadowData.at(nullptr).GetData(m_shadowingLightHandle.GetIndex()).m_shadowFilterMethod;
```

Comment:
minor: this should probably be checked beforehand and a reference saved to it in a variable since it's accessed multiple times